qutebrowser/misc/sessions.py

```diff
@@ -413,10 +413,14 @@ def _load_window(self, win):
                                     window=window.win_id)
         tab_to_focus = None
         for i, tab in enumerate(win['tabs']):
-            new_tab = tabbed_browser.tabopen(background=False)
-            self._load_tab(new_tab, tab)
             if tab.get('active', False):
                 tab_to_focus = i
+            background = tab_to_focus != i
+            new_tab = tabbed_browser.tabopen(background=background)
+            self._load_tab(new_tab, tab)
+            new_tab.show()
```

Comment:
Maybe I should add a WORKAROUND comment here to clarify why we are managing tab focus at all and doing it manually instead of letting the tab bar do it.